src/components/API/getMenuAPI.jsx

```diff
@@ -0,0 +1,12 @@
+import {React, useState} from 'react';
```

Comment:
It seems they were probably left in by mistake. If they're not necessary, I recommend removing them.